src/backend/core/mda/outbound.py

```
@@ -236,7 +236,36 @@ def _mark_delivered(
             internal: bool,
             error: Optional[str] = None,
             retry: Optional[bool] = False,
+            smtp_context: SMTPMailContext = {}
```

Ruff (0.13.1)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)